### PR TITLE
Buddha's Sayings

### PR DESCRIPTION
**Pull request by JJM317**

### CODE REVIEW

File: _content/monographs/buddhas-sayings_ireland.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - sutta
+year: 1991
+olid: OL13056106M
+oclc: 35595327
+publisher: bps
+address: "Kandy"
+pages: 65
+---
+
+This is John D. Ireland's 1991 translation of Itivuttaka. This collection of 112 suttas, attributed to Khujjuttara's (a female lay disciple of the Buddha) recollection of the Buddha's teaching, is located in the Khuddaka Nikāya, which itself is found in the larger Sutta Piṭaka of the Pāli Canon.
```
Comment (khemarato) [inline _content/monographs/buddhas-sayings_ireland.md:20]: A bit generic. I'd talk about the poetry of this particular translation.

File: _content/monographs/buddhas-sayings_ireland.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title: "The Itivuttaka: Buddha's Sayings"
+authors: ireland
```
Comment (khemarato) [inline _content/monographs/buddhas-sayings_ireland.md:3]: `authors` is an array. It needs `[]` around it :smile:

File: _content/monographs/buddhas-sayings_ireland.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+external_url: "https://readingfaithfully.org/itivuttaka-the-buddhas-sayings-translated-by-john-d-ireland-epub-kindle-pdf/"
+drive_links:
+  - "https://drive.google.com/file/d/12Tjx_JKzmxdMChWMt0YM_L1vA1jSQtiN/view?usp=drive_link"
+  - "https://drive.google.com/file/d/1Lh56kURtwqss5NDOwd08S4q03GVaszif/view?usp=drive_link"
+  - "https://drive.google.com/drive/folders/1C3T-Xh7fk0ynlQvuMmw3O68j7RPGs-eT?usp=drive_link"
+formats: [pdf, epub, mp3]
```
Comment (khemarato) [inline _content/monographs/buddhas-sayings_ireland.md:9]: MP3s (plural)

**Review by khemarato — COMMENTED** [review]
In this case, the Netlify build caught the `authors` error :smile_cat:

File: _content/monographs/buddhas-sayings_ireland.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+title: "The Itivuttaka: Buddha's Sayings"
+authors: ireland
+external_url: "https://readingfaithfully.org/itivuttaka-the-buddhas-sayings-translated-by-john-d-ireland-epub-kindle-pdf/"
+drive_links:
+  - "https://drive.google.com/file/d/12Tjx_JKzmxdMChWMt0YM_L1vA1jSQtiN/view?usp=drive_link"
+  - "https://drive.google.com/file/d/1Lh56kURtwqss5NDOwd08S4q03GVaszif/view?usp=drive_link"
```
Comment (khemarato) [inline _content/monographs/buddhas-sayings_ireland.md:7]: https://drive.google.com/file/d/1VNxFhCTPv5RNL9UG7Wbx9PYJlrEtQI5Z/view?usp=drivesdk

File: _content/monographs/buddhas-sayings_ireland.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+---
+title: "The Itivuttaka: Buddha's Sayings"
+authors: ireland
+external_url: "https://readingfaithfully.org/itivuttaka-the-buddhas-sayings-translated-by-john-d-ireland-epub-kindle-pdf/"
+drive_links:
+  - "https://drive.google.com/file/d/12Tjx_JKzmxdMChWMt0YM_L1vA1jSQtiN/view?usp=drive_link"
```
Comment (khemarato) [inline _content/monographs/buddhas-sayings_ireland.md:6]: https://drive.google.com/file/d/1FTB4QFBGNxVGoVAcQs-j9BdY-piZuMBw/view?usp=drivesdk

File: _content/monographs/buddhas-sayings_ireland.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+  - "https://drive.google.com/file/d/1Lh56kURtwqss5NDOwd08S4q03GVaszif/view?usp=drive_link"
+  - "https://drive.google.com/drive/folders/1C3T-Xh7fk0ynlQvuMmw3O68j7RPGs-eT?usp=drive_link"
```
Comment (khemarato) [inline _content/monographs/buddhas-sayings_ireland.md:8]: https://drive.google.com/drive/folders/1C3T-Xh7fk0ynlQvuMmw3O68j7RPGs-eT

**Review by khemarato — COMMENTED** [review]
Please use the new drive links

File: _content/monographs/buddhas-sayings_ireland.md (hunk 2 above)
Comment (JJM317) [reply _content/monographs/buddhas-sayings_ireland.md:3]: I didn't understand this fix. Maybe you can explain it when we meet Sunday.

**Pull request merged**